cpp/src/arrow/util/hash-util.h

```diff
@@ -67,14 +76,37 @@ class HashUtil {
     return hash;
   }
 
+  static uint32_t CrcHashARMCE(const void* data, int32_t nbytes, uint32_t hash) {
```

Comment:
Is it possible to reconcile the two functions instead of duplicating the code with different function names?
Same below for `DoubleCRCHash`.

Reply:
Yes, it makes sense.